scripts/pkg/build_templates/opensearch/deb/debmake_opensearch_install.sh

```diff
@@ -41,6 +41,6 @@ ln -s ${data_dir} ${buildroot}${product_dir}/data
 ln -s ${log_dir}  ${buildroot}${product_dir}/logs
 
 # Change Permissions
-chmod -Rf a+rX,u+w,g-w,o-w ${buildroot}/*
+chmod -Rf u=rwX,g=rX,o=rX ${buildroot}/*
```

Comment:
Do we want to have this line here as well, just for the sake of consistency?
```
chmod -Rf g-s %{buildroot}/*
```

Reply:
From what I see, the packaging does not copy the SGID bit when installing, so I think we can skip it.

Reply:
Ah, maybe when installing from rpm it is needed.  I cannot test ATM, but will be able next week.

Reply:
Yes, I am more used to Debian and the .deb does not set the SGID bit.  Either it is not copied in the package, or ignored when installing.  I'll add the explicit removal so that the .rpm package is in sync with the Debian one and with what I would expect for a package.